internal/cluster/validations/validations.go

```diff
@@ -175,13 +176,20 @@ func ValidateClusterNameFormat(name string) error {
 
 // ValidateHTTPProxyFormat validates the HTTP Proxy and HTTPS Proxy format
 func ValidateHTTPProxyFormat(proxyURL string) error {
-	if strings.HasPrefix(proxyURL, "https") {
+	if !govalidator.IsURL(proxyURL) {
+		return fmt.Errorf("Proxy URL format is not valid: '%s'", proxyURL)
+	}
+	u, err := url.Parse(proxyURL)
+	if err != nil {
+		return fmt.Errorf("Proxy URL format is not valid: '%s'", proxyURL)
+	}
+	if u.Scheme == "https" {
```

Comment:
Don't think this is required due to line 189

Reply:
no, according to the documentation [1], the description of the httpsProxy is:

> A proxy URL to use for creating HTTPS connections outside the cluster. If this is not specified, then httpProxy is used for both HTTP and HTTPS connections. The URL scheme must be http; https is currently not supported.

I would like to have a specific message for this case.

[1] https://docs.openshift.com/container-platform/4.5/networking/enable-cluster-wide-proxy.html#nw-proxy-configure-object_config-cluster-wide-proxy